static/src/stylesheets/module/_release-message.scss

```diff
@@ -939,3 +978,105 @@ $btn-height: gs-height(1) - $gs-baseline/2; //30px
     width: 100%;
     height: 100%;
 }
+
+.membership__remind-me-form {
+    border-top: 1px dotted #333333;
+    margin-top: 12px;
+    width: 100%;
+}
+
+.membership__engagement-text-field {
+    @include clearfix;
+    @include circular;
+    padding-left: $gs-gutter;
+    color: black;
+    @include mq($from: mobile) {
+        font-size: 12px;
+        line-height: 24px;
+        height: 36px;
+        margin-top: 12px;
+    }
+    @include mq($from: tablet) {
+        font-size: 14px;
+        line-height: 28px;
+        height: 36px;
+    }
+    @include mq($from: desktop) {
+        font-size: 14px;
+        line-height: 36px;
+        height: 36px;
+        width: 269px;
+    }
+    position: relative;
+    border: 0;
+    width: 100%;
+}
+
+.message-button-rounded__cta.membership__remind-me-form__cta {
+    color: #fffffb;
+    background-color: #333333;
+    top: 0px;
+    z-index: $zindex-popover + 1;
+    margin-top: 6px;
+
+    @include mq($from: desktop) {
+        margin-left: 10px;
+        margin-top: 12px;
+    }
+
+}
+
+.membership__remind-me-form {
+
+    .membership__remind-me-form__error {
+        @include fs-textSans(2);
+        font-size: 12px;
+        line-height: 16px;
+        position: relative;
+        margin-top: 4px;
+        display: block;
+    }
+
+    .membership__remind-me-form__error.yellow {
+        color: #b51800;
+    }
+
+    .membership__remind-me-form__error.bright-blue {
+        color: #b51800;
+    }
+
+    .membership__remind-me-form__error.purple {
+        color: #ffffff;
+    }
+
+    .membership__remind-me-form__error.dark-blue {
+        color: #ffffff;
+    }
+    .membership__remind-me-form__thanks-message {
+
+    }
+
+    .membership__remind-me-form__error.hide-element,
+    .membership__remind-me-form__thanks-message.hide-element {
+        display: none;
+    }
+
+    .membership__remind-me-form__thanks-message {
+        margin-top: 12px;
```

Comment:
Could we switch some of the px to values from [the Grid](https://github.com/guardian/guss-grid-system)?

e.g. `margin-top: $gs-baseline`